pytorch_lightning/trainer/connectors/callback_connector.py

```diff
@@ -107,3 +110,20 @@ def attach_model_logging_functions(self, model):
         for callback in self.trainer.callbacks:
             callback.log = model.log
             callback.log_dict = model.log_dict
+
+
+def _tail_checkpoint_callbacks(callbacks: List[Callback]) -> List[Callback]:
```

Comment:
I'm looking for a better name here, concisely describing that we want to push all checkpoints to the end. 
Not sure if "tail" is the right expression here...

Reply:
how about `_reorder_callbacks` in case there are other steps we want to add here later? can't think of anything off the top of my head though. maybe in the future there are callback DAGs and this is where we top sort them 😄